roles/bootstrap/tasks/root_tasks.yml

```diff
@@ -20,6 +20,13 @@
     - { name: 'fs.file-max', value: 1000000 }
   when: tuning_kernel_parameters
 
+- name: disable THP
+  shell: echo never > /sys/kernel/mm/transparent_hugepage/enabled && echo never > /sys/kernel/mm/transparent_hugepage/defrag
+  when:
+    - ansible_os_family == 'RedHat'
```

Comment:
can we use in ubuntu?

Reply:
Yes, already remove operation system limit.